### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by ashishkatiyar2404**
Screenshot (821)
Screenshot (822)
Screenshot (823)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+  margin: 1.6rem;
+}
+
+.btn__white {
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 1.5rem 4rem;
+  display: inline-block;
+  border-radius: 10rem;
+
```
Comment (pradipgharge) [inline style.css:146]: remove extra spacing here

File: index.html [hunk 2]
```diff
@@ -4,8 +4,129 @@
...
+    </header>
+    <!-- Nav bar end -->
+
+    <!-- SECTION 1 -->
+    <section class="main__section">
+      <div class="heading__text-box">
+        <h1 class="heading__primary">
+          <span class="heading__primary-main">Welcome to Music Store! </span>
+          <span class="heading__primary-sub">
+            <!-- At our guitar store you'll be able to search for huge range of
```
Comment (pradipgharge) [inline index.html:51]: Clear commented code before pushing.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+  width: 100%;
+}
+
+.new {
+  width: fit-content;
+  padding: 0.5rem;
+  background-color: var(--color-new-badge);
+  border-radius: 2px;
+  position: absolute;
+  top: 0%;
```
Comment (pradipgharge) [inline style.css:227]: % is not needed in my opinion, you can leave it px.

**Review by pradipgharge — APPROVED** [review]
The code looks good.
 Some comments on UI - 
 1) Add a border to the search bar.
 2)Line break after featured products is partial ( not sure if its intentionally done that way).
 3)Add a footer.

File: index.html [hunk 4]
```diff
@@ -4,8 +4,129 @@
...
+          <img src="/assets/images/featured__drum.jpg" alt="Drums" />
+          <span class="featured__item-name drum">Drums</span>
+        </div>
+        <div class="featured__xylo">
+          <img src="/assets/images/featured__xylo.jpg" alt="Xylophones" />
+          <span class="featured__item-name xylo">Xylophones</span>
+        </div>
+      </div>
+    </section>
+    <hr />
```
Comment (ShrutiLakhani) [inline index.html:87]: Please avoid using <hr> & <br> tags in html and try to add the functionality in CSS

File: style.css [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+  font-weight: 900;
+  letter-spacing: 1rem;
+}
+
+.btn__white {
+  background-color: var(--color-white);
+  color: var(--color-gray);
+  margin: 1.6rem;
+}
+
```
Comment (ShrutiLakhani) [inline style.css:139]: Seems redundant. Please chk

File: style.css [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+.btn__white {
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 1.5rem 4rem;
+  display: inline-block;
+  border-radius: 10rem;
```
Comment (ShrutiLakhani) [inline style.css:145]: Border radius 10 rem seems too much

**Review by ShrutiLakhani — APPROVED** [review]
Suggested some changes. Please look into those.

File: style.css (hunk 3 above)
Comment (ashishkatiyar2404) [reply style.css:227]: ok

File: index.html (hunk 2 above)
Comment (ashishkatiyar2404) [reply index.html:51]: sure... this one i forgot to remove

File: index.html (hunk 4 above)
Comment (ashishkatiyar2404) [reply index.html:87]: ok

**Pull request merged**